docs/src/components/ComponentPage/ComponentPage.js

```diff
@@ -89,23 +89,27 @@ const ComponentPage = (props: Props) => {
           </BpkList>
         )}
 
-        <BpkText
-          tagName="h3"
-          textStyle={TEXT_STYLES.sm}
-          weight={WEIGHT_STYLES.bold}
-          className={getClassName(
-            'bpkdocs-component-page__table-of-contents-heading',
-          )}
-        >
-          Examples
-        </BpkText>
-        <BpkList>
-          {examples.map(example => (
-            <BpkListItem key={example.id}>
-              <BpkLink href={`#${example.id}`}>{example.title}</BpkLink>
-            </BpkListItem>
-          ))}
-        </BpkList>
+        {examples.length > 0 && (
```

Comment:
Had to do this because there is one page — `IOSThemingPage` — that doesn't have any screenshots (just a readme). I think we can fix this by giving it some screenshots to show an example of theming, but I did this as a quick fix for now.